compiler-plugin/src/main/scala/com/trueaccord/scalapb/compiler/ProtobufGenerator.scala

```diff
@@ -1182,7 +1182,7 @@ class ProtobufGenerator(val params: GeneratorParams) extends DescriptorPimps {
       .call(generateMessageCompanion(message))
   }
 
-  def scalaFileHeader(file: FileDescriptor): FunctionalPrinter = {
+  def scalaFileHeader(file: FileDescriptor, javaConverterImport: Boolean = true): FunctionalPrinter = {
```

Comment:
can you make this not have a default value and change all callers to either pass file.javaConversions or false.